topic_dynamics/tests/test_parsing.py

```diff
@@ -0,0 +1,37 @@
+"""
+Parsing-related tests.
+"""
+
+import os
+import unittest
+
+from ..parsing import get_identifiers
+
+tests_dir = os.path.abspath(os.path.dirname(__file__))
+
+
+class TestParser (unittest.TestCase):
+    def test_java(self):
```

Comment:
all tests follow the same logic - make a function with this logic and call with different arguments in each tests

Reply:
Couldn't make the calls of functions with "self", will ask you later.